### PR TITLE
IGAPP-708: Noto Sans

### PR DESCRIPTION
**Pull request by f1sh1918**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

This PR replaces **Open-Sans** with **Noto Sans** and **Lateef** with **Noto Sans Arabic**.
Please check if the fonts gonna be displayed according to the font-config.

**Example Integreat Font Config:**

```js
const fonts: FontsType = {
  native: {
    decorativeFontBold: 'Raleway-Bold',
    decorativeFontRegular: 'Raleway-Regular',
    contentFontRegular: 'NotoSans-Regular',
    contentFontBold: 'NotoSans-Bold',
    webviewFont: 'Noto Sans, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif'
  },
  web: {
    decorativeFont: 'Raleway, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif',
    contentFont: 'Noto Sans, Noto Sans Arabic, Noto Sans SC, Noto Sans Georgian, sans-serif'
  },
  ...commonFonts
}
```

### CODE REVIEW

**Review by klinzo — CHANGES_REQUESTED** [review]
image :eyes: 

Tested on firefox and working fine. However, the native fonts are missing, therefore it is not working on iOS and probably only working on android because noto sans is the standard google/android font.

File: web/www/fonts/open-sans/LICENSE.txt [hunk 1]
```diff
@@ -1,202 +0,0 @@
-
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply web/www/fonts/open-sans/LICENSE.txt:1]: noto sans arabic from google fonts did not provide any license. i'm gonna check this

File: web/www/fonts/noto-sans/noto-sans.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/* noto-sans-regular - vietnamese_latin-ext_latin_greek-ext_greek_devanagari_cyrillic-ext_cyrillic */
+@font-face {
+  font-family: 'Noto Sans';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Noto Sans Regular'), local('NotoSans-Regular'),
```
[the remark this answers is not in the document]
Comment (f1sh1918) [reply web/www/fonts/noto-sans/noto-sans.css:6]: yes this is the standard way to check if the user has the font files locally installed. Recommend way and also standard generation of these font css file. But the correct way is to check for 'Noto Sans' , 'NotoSans-Regular' :)
This is recommended for often used fonts

File: web/www/fonts/open-sans/LICENSE.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (f1sh1918) [reply web/www/fonts/open-sans/LICENSE.txt:1]: it's also SIL Open Font License. I added the file.
https://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL

**Comment by f1sh1918** [comment]
> image 👀
> 
> Tested on firefox and working fine. However, the native fonts are missing, therefore it is not working on iOS and probably only working on android because noto sans is the standard google/android font.

Added font files and changed adjusted font-families since IOS is not using filename but font names (f.e. NotoSans)
@klinzo @SteffiStuffel 
Can you please test the fonts on ios again.
I dunno how to inspect fonts in webview. It looks fine but i can not check that properly

**Review by Taggotty — APPROVED** [review]
Tested on web and android

File: native/src/utils/renderHtml.ts [hunk 3]
```diff
@@ -118,10 +118,16 @@ const renderHtml = (
...
+      font-family: 'Noto Sans Arabic';
       font-style: normal;
       font-weight: 400;
-      src: ${getFontFaceSource('Lateef')};
+      src: ${getFontFaceSource('NotoSansArabic-Regular')};
+    }
+    @font-face {
+      font-family: 'Noto Sans Arabic';
+      font-style: normal;
+      font-weight: 400;
```
Comment (klinzo) [inline native/src/utils/renderHtml.ts:129]: I guess:
```suggestion
      font-weight: 700;
```

File: native/ios/Integreat/Info.plist [hunk 4]
```diff
@@ -45,9 +45,10 @@
 	</dict>
 	<key>UIAppFonts</key>
 	<array>
-		<string>Lateef.ttf</string>
-		<string>OpenSans-Bold.ttf</string>
-		<string>OpenSans-Regular.ttf</string>
+		<string>NotoSansArabic-Regular.ttf</string>
+			<string>NotoSansArabic-Bold.ttf</string>
```
Comment (klinzo) [inline native/ios/Integreat/Info.plist:49]:
```suggestion
		<string>NotoSansArabic-Bold.ttf</string>
```

File: native/ios/Integreat.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -214,6 +220,11 @@
 		A330D32D22AA9F42005736D0 /* fonts */ = {
 			isa = PBXGroup;
 			children = (
+				2E0440FB26C133EE00183828 /* NotoSans.ttf */,
+				2E0440F426C127AF00183828 /* NotoSans-Bold.ttf */,
+				2E0440F526C127B000183828 /* NotoSans-Regular.ttf */,
```
Comment (klinzo) [inline native/ios/Integreat.xcodeproj/project.pbxproj:225]: Any reason why we need `NotoSans.ttf` and NotoSans-Regular.ttf`? I think we have just `NotoSans.ttf`.

Comment (f1sh1918) [reply native/ios/Integreat.xcodeproj/project.pbxproj:225]: No i think it was not removed properly even i removed it in xcode

File: native/src/utils/renderHtml.ts (hunk 3 above)
Comment (f1sh1918) [reply native/src/utils/renderHtml.ts:129]: first one is regular 400 weight, second one is bold with 700. For me everything should be fine

**Comment by klinzo** [comment]
Tested on iOS, works fine! I think a release note would be nice for this :)

**Review by klinzo — APPROVED** [review]

**Pull request merged**